### PR TITLE
Feat sidebar

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
I have added a sidebar for the video library. Using the sidebar we can traverse different pages of the library using routes easily.

### CODE REVIEW

File: src/pages/VideoListing/VideoListing.jsx [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
-        </div>
-    );
-}
\ No newline at end of file
+import React from "react";
+import { GetVideo } from "../../context/context";
+import { VideoCard } from "../../components/components";
+
+export default function VideoListing() {
+  const videoList = GetVideo().videos;
```
Comment (JantuDeb) [inline src/pages/VideoListing/VideoListing.jsx:6]: ` const {videos}= GetVideo();`

**Review by JantuDeb — APPROVED** [review]
This is looking good

File: src/pages/VideoListing/VideoListing.jsx (hunk 1 above)
Comment (Kothalkarpriya) [reply src/pages/VideoListing/VideoListing.jsx:6]: Thanks, I will do it

**Review by harshalini — APPROVED** [review]
Code is clean, looks good to merge

**Pull request merged**